packages/server/graphql/mutations/endRetrospective.ts

```diff
@@ -151,7 +151,10 @@ export default {
     if (!isTeamMember(authToken, teamId) && authToken.rol !== 'su') {
       return standardError(new Error('Team not found'), {userId: viewerId})
     }
-    if (endedAt) return standardError(new Error('Meeting already ended'), {userId: viewerId})
+    if (endedAt) {
+      await sendNewMeetingSummary(meeting, context).catch(console.log)
```

Comment:
Just for debugging, allows to run
```
mutation EndRetro {
  endRetrospective(meetingId: "nHMYJziEmc") {
    __typename
  }
}
```
repeatedly on the same meeting to regenerate/resend the summary.